deployment/aws/package-lock.json

```diff
@@ -0,0 +1,1436 @@
+{
```

Comment:
@kylebarron I don't think we have to add the package-lock here 🤷‍♂️. having package-lock will mean that the contributor has to run npm install before pushing to the repo (in case of change of cdk dependency). I think it's a step we can skip ?

Reply:
I think it's fine to remove. Hopefully AWS never makes a broken cdk release 😄